test/jestExportsMapResolver.cjs

```diff
@@ -1,11 +1,27 @@
 // Temporary workaround for https://github.com/facebook/jest/issues/9771
 // Source: https://github.com/facebook/jest/issues/9771#issuecomment-841624042
 
-const resolver = require("enhanced-resolve").create.sync({
-  conditionNames: ["node", "require", "default"],
-  extensions: [".js", ".json", ".node", ".ts"],
-});
+const enhancedResolve = require("enhanced-resolve");
+
+const EXTENSIONS = [".js", ".json", ".node", ".ts"];
+
+const resolvers = new Map();
+function getResolver(conditionNames) {
+  const key = conditionNames.join(":::");
```

Comment:
Q: Is ":::" a hardcoded separator used by `enhanced-resolve`? Can we link to where it comes from?

Reply:
It's just to build a cache key, to avoid reinstantiating the resolver for every request. Should I rename the map to `resolversCache`?

Reply:
Either looks good to me, I would prefer name the `":::"` actually, to signal that it is only used by us internally.